app/javascript/packages/verify-flow/verify-flow-alert.tsx

```diff
@@ -0,0 +1,30 @@
+import { t } from '@18f/identity-i18n';
+import { Alert } from '@18f/identity-components';
+
+interface VerifyFlowAlertProps {
+  /**
+   * Current step name.
+   */
+  currentStep: string;
+}
+
+function VerifyFlowAlert({ currentStep }: VerifyFlowAlertProps) {
+  let message;
+  switch (currentStep) {
+    case 'personal_key':
+    case 'personal_key_confirm':
+      message = t('idv.messages.confirm');
+      break;
+
+    default:
+      return null;
+  }
```

Comment:
With so few cases, not sure switch/case makes the most sense here, but if we're planning for the future I'd recommend extracting into a function so we can just return always instead of having to remember to assign to `message` and also remember `break`? (I'd probably really extract it instead of IIFE but this is the gist of it)

```suggestion
  const message = ((currentStep) => {
    switch (currentStep) {
      case 'personal_key':
      case 'personal_key_confirm':
        return t('idv.messages.confirm');
      default:
        return null;
      }
    })(currentStep);
    
  if (!message) {
    return null;
  }
```

Reply:
Yeah that's a fair point. I agree that early returns are easier to follow than a `switch`. Also, the `default: return null;` is kinda easy to overlook, but our linting enforces the `default` block, so I figured it was a decent use of it. Probably rather just disable that lint rule, honestly.

Reply:
Oh, I missed that your point was to use early returns in combination with `switch`. I was thinking just get rid of the `switch` altogether, and have a series of `if`s instead, e.g.

```ts
if (step === '...') {
  return '...';
}

if (step === '...') {
   return '...';
}
```

Dunno if you have much preference one way or the other.

Reply:
I extracted a `getStepMessage` in 86bc1d8. I expect this will grow a bit, since we're aiming for feature parity, and there's other similar messages in the flow.

>We have found records matching your Phone of record

>We have verified your personal information

>We have verified your state-issued ID

etc.